src/cellviz.cpp

```
@@ -32,171 +57,198 @@
 
 int main(int argc, char *argv[])
 {
+    while(true){
+        gtk_init(&argc, &argv);
```

cppcheck (2.10-2)

[performance] Variable 'species' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)